lib/tasks/truncate_csv_and_loader_tables.rake

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+desc 'truncate HUD loader csv and importer tables'
+# rails truncate_csv_and_loader_tables[2022]
+# rails truncate_csv_and_loader_tables[2022,confirmed]
+task :truncate_csv_and_loader_tables, [:year, :confirm] => :environment do |_task, args|
+  year = args.year
+  raise 'valid year required' unless year =~ /\A202\d\z/
+
+  connection = HmisCsvImporter::Importer::ImporterLog.connection
+  tables = connection.tables.
```

Comment:
This seems slightly dangerous to me since we might have other tables that contain `hmis_#{year}`.  Can we lean on `drivers/hmis_csv_twenty_twenty/app/models/hmis_csv_twenty_twenty.rb#importable_files_map` and `drivers/hmis_csv_twenty_twenty_two/app/models/hmis_csv_twenty_twenty_two.rb#importable_files_map` instead?

That would also prevent accidental truncation of 2024 tables.  Probably best to have a case statement that determines the namespace based on the year passed in.